db/migrate/20250820145646_add_psd_enabled_to_sessions.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddPsdEnabledToSessions < ActiveRecord::Migration[8.0]
+  def change
+    add_column :sessions, :psd_enabled, :boolean, default: true, null: false
```

Comment:
```suggestion
    add_column :sessions, :psd_enabled, :boolean, default: false, null: false
```

Actually I'm wondering if it makes sense to start with this disabled, to avoid accidentally enabling PSD when we shouldn't.